examples/thesis/thesis.md

```diff
@@ -0,0 +1,70 @@
+```config
+lang = "en"
+document_type = "thesis"
+title = "Fancy Title"
+subtitle = "Subtitle (e.g. title in native language)"
+author = "My Name"
+date = "\\today"
+supervisor = "My Supervisor"
+advisor = "My Advisor"
+logo_university = "https://upload.wikimedia.org/wikipedia/commons/thumb/4/48/Markdown-mark.svg/40px-Markdown-mark.svg.png"
+logo_faculty = "https://upload.wikimedia.org/wikipedia/commons/thumb/9/92/LaTeX_logo.svg/40px-LaTeX_logo.svg.png"
+university = "Univeristy of Duckburg"
+faculty = "Department of Quak"
+thesis_type = "Bachelor's Thesis in Quak"
+location = "Duckburg"
+disclaimer = "I confirm that this master's thesis is my own work and I have documented all sources and material used."
```

Comment:
This is supposedly a `Bachelor's Thesis in Quak`, inconsistent with the disclaimer.